packages/stripe_android/android/build.gradle

```diff
@@ -38,10 +38,10 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.3.1"
+    implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.4.1"
```

Comment:
are these dependency updates on purpose? Otherwise, I'd follow the versions from stripe-react-native

Reply:
yeah I thought upgrading those minors couldn't hurt but no problem reverting this. However I would like to upgrade the gradle stuff since it gets rid of a lot of warnings and also the Flutter team recommends these versions now